Source/WebCore/css/MediaQueryEvaluator.cpp

```diff
@@ -481,6 +481,33 @@ static bool dynamicRangeEvaluate(CSSValue* value, const CSSToLengthConversionDat
     }
 }
 
+static bool scanEvaluate(CSSValue* value, const CSSToLengthConversionData&, Frame& frame, MediaFeaturePrefix)
+{
+    auto view = frame.view();
```

Comment:
My understanding is that all ref-counted stack variables in new code should be protected by `Ref` or `RefPtr`.

So this would become `RefPtr view = frame.view()`.